lists/global.csv

```diff
@@ -67,6 +68,7 @@ http://creativecommons.org,NEWS,News Media,2014-04-15,citizenlab,Updated by OONI
 https://cryptome.org,NEWS,News Media,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 http://cultdeadcow.com,HACK,Hacking Tools,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
 https://cyber.harvard.edu/,NEWS,News Media,2014-04-15,citizenlab,Updated by OONI on 2017-02-14
+https://dashjr.org/,COMM,E-commerce,2018-11-15,blockchain,Initial tracking of cryptocurrencies on 2018-11-15
```

Comment:
These are bootstrapping addresses hardcoded into the bitcoin source code: https://github.com/bitcoin/bitcoin/blob/863e995b79ec388bf292d80f181912d01e20e2e5/src/net.cpp#L1128